### PR TITLE
1.12/1686 bug error responses not being returned to bad requests

### PR DESCRIPTION
**Pull request by charlesstrange2**
Updated some blades to add the error boxes
added some tests

https://trello.com/c/eHWdtqcR/1686-bug-error-responses-not-being-returned-to-bad-requests

### CODE REVIEW

File: app/Http/Requests/AdminNewUpdateMarketRequest.php [hunk 1]
```diff
@@ -24,9 +25,21 @@ public function authorize()
      */
     public function rules()
     {
+        // Get the ID off the route
+        $id = $this->route('id') ?? null;
+        $sponsor_id = $this->only('sponsor');
```
Comment (AlinChindea) [inline app/Http/Requests/AdminNewUpdateMarketRequest.php:30]: does the only method plucks the id of whatever var name we're passing in there? nice

File: resources/views/service/traders/edit.blade.php [hunk 2]
```diff
@@ -72,7 +72,7 @@ class="styled-checkbox {{ $errors->has('disabled') ? 'error' : '' }}"
                     </div>
                 </div>
                 @include('service.partials.addTraderUser', ['users' => $trader->users ])
-                <button type="submit" id="updateMarket">Update All</button>
+                <button type="submit" id="updateTrader">Update All</button>
```
Comment (AlinChindea) [inline resources/views/service/traders/edit.blade.php:75]: oops 😄

File: app/Http/Requests/AdminNewUpdateMarketRequest.php (hunk 1 above)
Comment (AlinChindea) [reply app/Http/Requests/AdminNewUpdateMarketRequest.php:30]: if it's the Laravel collections method only - https://laravel.com/docs/6.x/collections#method-only - i think your argument here should be `sponsor_id` if you need the id of the sponsor. same for market in the AdminNewUpdateTraderRequest

Comment (charlesstrange2) [reply app/Http/Requests/AdminNewUpdateMarketRequest.php:30]: `$this` refers to the formRequest, which is a type of Request. only, will pull the input off. 
However, you may be right, i don't think i want an array, which only returns.

**Review by AlinChindea — APPROVED** [review]
🐙 okey dokey

**Pull request merged**